citrination_client/search/dataset/query/dataset_returning_query.py

```diff
@@ -3,7 +3,7 @@
 
 class DatasetReturningQuery(BaseReturningQuery):
     """
-    Query used to return information about datasets.
+    Query used to return information about datasets. Results limited to 50,000. Please chose values for from_index and size that do not exceed this limit.
```

Comment:
Don't include the 50k number in here. That's asking to become out of date.

Reply:
In principle, I agree with you. However, this is a common issue for users and we had a specific request that it be documented. Once we add an API endpt that can return this value, we can reconsider but for now I would weight more heavily communicating correctly over avoiding risk of obsolescence.